primitives/src/jobs/mod.rs

```diff
@@ -229,32 +240,35 @@ pub enum ValidatorOffence {
 
 #[derive(PartialEq, Eq, Encode, Decode, RuntimeDebug, TypeInfo, Clone)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub struct RpcResponseJobsData<AccountId> {
+pub struct RpcResponseJobsData<AccountId, BlockNumber> {
 	/// The job id of the job
 	pub job_id: JobId,
 
 	/// The type of the job submission.
 	pub job_type: JobType<AccountId>,
 
-	/// (Optional) List of participants' account IDs.
-	pub participants: Option<Vec<AccountId>>,
```

Comment:
Without this how will the gadget get the list of participants for phase2 job? Or should it be a manual fetch using `get_phase_1_result`?

Reply:
Yup, exactly. We fetch the result. And now they all call Phase results, not phase one or two results.